packages/material-ui-lab/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -113,27 +124,25 @@ const DateTimePickerToolbar: React.FC<ToolbarComponentProps & WithStyles<typeof
               selected={openView === 'day'}
               value={dateText}
             />
-          </div>
-          <div className={classes.timeContainer}>
+          </DateTimePickerToolbarDateContainer>
+          <DateTimePickerToolbarTimeContainer>
             <PickersToolbarButton
               variant="h3"
               data-mui-test="hours"
               onClick={() => setOpenView('hours')}
               selected={openView === 'hours'}
               value={date ? formatHours(date) : '--'}
-              typographyClassName={classes.timeTypography}
```

Comment:
Did you check where this class was used? Just removing it can break some scenarios, if it is used as a selector.

Reply:
Looks like is harmless and was added only for customization... Let's keep the change

Reply:
yeah, `timeTypography: {}` does nothing here